### DISCUSSION TITLE
first-time.php missing have we moved on to sso setup?

### DISCUSSION THREAD

**Discussion by njhughes-01** [openthc/docker#6] [Q&A] [answered]
Are the files for CRE key generation present, or is this still in the to-do pile? I have tried using the first-time.php, but that file doesn't appear to exist.  

```
sudo docker-compose exec app php ./sbin/first-time.php
Could not open input file: ./sbin/first-time.php

```
What is the best way to manually generate a key pair to get me off the ground?

I would like to get self-hosted running as I am learning the platform before I apply for my own commercial in my state or recommend this to my clients I work with.

Any guidance is much appreciated.

**Comment by djbusby** [comment]
The 'first-time.php' script is no longer needed and, at the moment these docker images are out-dated.  We will get an updated published.  There will be updated instructions here as well.

**Comment by djbusby** [comment, marked as answer]
The docker images have been updated and the references to the un-used first-time.php script have been removed.
